run_fvs.sh

```diff
@@ -6,17 +6,34 @@ date
 pwd
 git status
 
-./build_node.sh
-./create_binary.sh
+docker rm -f host1 || true
+docker run --privileged -v `pwd`:/code --name host1 -tid jpetazzo/dind
+
+docker exec -t host1 bash -c \
+ 'docker -d & \
```

Comment:
Yep. If I take out only that, it says;
`FATA[0000] Post http:///var/run/docker.sock/v1.18/build?cpusetcpus=&cpushares=0&dockerfile=Dockerfile&memory=0&memswap=0&rm=1&t=calico%2Fnode: dial unix /var/run/docker.sock: no such file or directory. Are you trying to connect to a TLS-enabled daemon without TLS?`